docs/schemas/project-registration-bican/DataCollection-metadata.csv

```diff
@@ -0,0 +1,16 @@
+﻿Field,DataType,Required,Description
+name,String,Y,"Human readable, locally unique identfier"
+title, String,Y,"Long form title of the collection"
+shortTitle, String,Y,"Short form title of the collection. Should be <30 characters"
+description, String,Y,"Description of the collection. Specify license application here."
+dataArchive,[Organization],Y,"Archive at which data will be deposited"
+species,[enum],Y,"Common name of species of donor from which data were collected. See species controlled vocabulary"
+accessControl,enum,Y,"Level of acess specified for data in collection. See accessControl controlled vocabulary"
+accessControlCode,enum,N,"Controlled vocabulary of codes that specify nature of access restriction. Values TBD"
+accessControlDescription,String,N,"Free text description of access restrictions"
+completionState,enum,Y,"Whether or not additional data is being generated for this collection. See completionState controlled vocabulary"
+modality,[enum],Y,"Defines modality of the data in this collection. See modality controlled vocabulary"
+technique,[enum],Y,"Technique used to acquire data in this collection. See technique controlled vocabulary"
+license,[enum],Y,"License applied to the data in this collection. See license controlled vocabulary"
+webResource,[String],N,"Links to relevant tools/pages to data collection"

```

Comment:
This is just a link; should we explicitly model the data repository and identifier for data repository?  For other tools for visualization or code, should we model these more explicitly?